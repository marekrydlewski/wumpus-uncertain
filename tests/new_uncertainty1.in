4 4
0.6
????
??B?
B???
OB??
