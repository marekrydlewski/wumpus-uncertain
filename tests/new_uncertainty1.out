0.60 0.60 0.62 0.60 
0.68 0.62 0.00 0.62 
0.00 0.81 0.62 0.60 
0.00 0.00 0.68 0.60 
